active/001-tabular-msa.md

```diff
@@ -0,0 +1,82 @@
+---
+Feature name: tabular-msa
+Start date: 2015-08-14
+RFC PR: <leave empty>
+scikit-bio issue: <leave empty>
+---
+
+# Summary
+
+Replace the current experimental `skbio.alignment.Alignment` class with a
+`TabularMSA` class, providing an API for manipulating tabular (row-column)
```

Comment:
I agree that having `MSA` at the end makes it read better and allows natural naming of other MSA classes (e.g., `PartialOrderMSA` and `ABruijnMSA`).

I like `MatrixMSA` but think `TabularMSA` is more descriptive of the structure and how users will interact with it. This will be a labeled matrix/array, with elements accessible via labels or positions, so it seems more table-like in nature than a standard matrix.